---
layout: hidden_page
hidden_title: BPPR Code
permalink: /bppr/
---
<div class="home">
Bash, Perl, Python, and R Code made by lab members for common use.<br>
<br>
<a href="https://github.com/TomJKono/Misc_Utils" rel="external">Miscellaneous Utilities</a> (Bash and Python) - by Tom Kono<br>
Tom’s utilities include Strip_BAM.sh, a shell script that strips BAM files down to a list of contigs that can be readily viewed in Consed.<br>
<br>
<a href="http://www.tc.umn.edu/~konox006/Code/SNPMeta/" rel="external">SNPMeta</a> (Python) - by Tom Kono<br>
It generates "metadata" for SNPs.<br>
<br>
<a href="http://faculty.agronomy.cfans.umn.edu/pmorrell/downloads/HudsontoFst.zip" rel="external">Hudson table to F statistic analysis</a> (Perl)<br>
The script converts SNP genotyping data in two line format, a format used for Richard Hudson's maxdip program and for libsequence tools, into genotype data appropriate for estimation of F statistics in heirfstat. Note that script deals with the genotype information, but to calculate F statistics the user will need to provide a population identifier for each sample.<br>
<br>
<a href="http://agronomy.cfans.umn.edu/pmorrell/downloads/rho_theta_plot.zip" rel="external">Custom Dot Plots</a> (R)<br>
We’ve used custom dot plots to depict point estimates and confidence intervals.
Figure 1 in Chen et al. 2008 (J of Heredity) is a nice example. The example code here has been updated by Ana Gonzales and is being used to depict estimates of recombination and mutation rate.<br>
<br>
<a href="https://github.com/MorrellLAB/sequence_handling" rel="external">Sequence Handling</a> (Bash) - by Paul Hoffman<br>
A pipeline to automate DNA sequence aligning and quality control workflows via list-based batch submission and parallel processing<br>
<br>
<a href="https://github.com/mojaveazure/angsd-wrapper" rel="external">ANGSD-wrapper</a> (Mixed) - by Paul Hoffman, Chaochih Liu, and Tyler Kent<br>
ANGSD-wrapper is a utility developed to aid in the analysis of next generation sequencing data.<br>
<br>
<a href="https://github.com/MorrellLAB/BAD_Mutations" rel="external">BAD_Mutations</a> (Python) - by Tom Kono<br>
BAD_Mutations (BLAST-Aligned-Deleterious?) performs a likelihood ratio test (LRT) for the prediction of deleterious variants, as described in Chun and Fay (2009). The package is designed to identify deleterious variants in plant species. BAD_Mutations is first used in Kono et al. (In Prep.), with publication detailing features of the software to follow.
</div>
