---
layout: page
title: Research
permalink: /research/
---

<div class="home">
<h2>The Role of Deleterious Mutations</h2>
<img src="{{ site.github_url }}{{ site.images_dir }}del_muts_F1.jpg" style="height: 200px;"><br>
Figure 1 from Kono et al. (2016). The Role of Deleterious Substitutions in Crop Genomes. MBE 33: 1669-1678<br>
<h4 style="font-size:150%;">Deleterious Mutations and the Response to Strong Selection in a Self-fertilizing Crop</h4>
Funded by the <a href="http://www.nsf.gov/" rel="external">National Science Foundation</a>.<br>
Co-PIs are <a href="http://www.genetics.wustl.edu/jflab/" rel="external">Justin Fay</a> and <a href="http://smithlab.cfans.umn.edu/" rel="external">Kevin Smith</a>.<br>
<br>
Durvasula A, Hoffman PJ, Kent TV, Liu C, Kono TJY, Morrell PL, Ross-Ibarra J (preprint). ANGSD-wrapper: utilities for analyzing next generation sequencing data. PeerJ PrePrints 3: e1837.<br>
• <a href="http://onlinelibrary.wiley.com/doi/10.1111/1755-0998.12578/full" rel="external">Journal publication</a> • <a href="https://peerj.com/preprints/1472/" rel="external">Preprint version</a> • <a href="https://github.com/mojaveazure/angsd-wrapper" rel="external">ANGSD-wrapper Github</a><br>
<br>
Liu Q, Zhou Y, Morrell PL, Gaut BS (preprint). Deleterious variants in Asian rice and the potential cost of domestication. bioRxiv.<br>
• <a href="http://biorxiv.org/content/early/2016/06/05/057224" rel="external">Preprint version</a><br>
<br>
Kono TJY, Fu F, Mohammadi M, Hoffman PJ, Liu C, Stupar RM, Smith KP, Tiffin P, Fay JC, Morrell PL (2016). The role of deleterious substitutions in crop genomes. Molecular Biology and Evolution 33: 1669-1678.<br>
• <a href="http://mbe.oxfordjournals.org/content/early/2016/06/23/molbev.msw102" rel="external">Journal publication</a> • <a href="http://biorxiv.org/content/early/2016/05/12/033175" rel="external">Preprint version</a> • <a href="https://github.com/MorrellLAB/BAD_Mutations" rel="external">BAD_Mutations Github</a><br>
<br>
Gaut BS, Diez CM, Morrell PL (2015). Genomics and the contrasting dynamics of annual and perennial domestication. Trends in Genetics 31: 709-719.<br>
• <a href="http://www.cell.com/trends/genetics/abstract/S0168-9525(15)00188-2" rel="external">Journal publication</a>

<h2>Biotechnology Risks</h2>
<img>
<h4 style="font-size:150%;">Comparison of mutation rates in soybean following transformation, mutagenesis, tissue culture and conventional breeding</h4>
Funded by the <a href="https://nifa.usda.gov/funding-opportunity/biotechnology-risk-assessment-research-grants-program-brag" rel="external">Biotechnology Risk Assessment Research Grants Program (BRAG)</a>.<br>
Awards page. <a href="http://cris.nifa.usda.gov/cgi-bin/starfinder/0?path=fastlink1.txt&id=anon&pass=&search=R=68661&format=WEBFMT6NT" rel="external">[Link 1]</a> <a href="http://cris.nifa.usda.gov/cgi-bin/starfinder/0?path=fastlink1.txt&id=anon&pass=&search=R=68661&format=WEBLINK" rel="external">[Link 2]</a><br>
Co-Project Directors are <a href="http://parrottlab.uga.edu/parrottlab/" rel="external">Wayne Parrott</a>, <a href="http://www.cropsoil.uga.edu/personnel/faculty/jackson.html" rel="external">Scott Jackson</a>, and <a href="http://stuparlab.cfans.umn.edu/" rel="external">Robert Stupar</a>.<br>
<br>
Anderson JE, Michno JM, Kono TJY, Stec AO, Campbell BW, Curtin SJ, Stupar RM (2016). Genomic variation and DNA repair associated with soybean transgenesis: a comparison to cultivars and mutagenized plants. BMC Biotechnology 16: 41.<br>
• <a href="https://bmcbiotechnol.biomedcentral.com/articles/10.1186/s12896-016-0271-z" rel="external">Journal Publication</a>
</div>

<!-- Start Google Analytics -->
<script type="text/javascript">
var _gaq = _gaq || [];
_gaq.push(['_setAccount', 'UA-19694768-1']);
_gaq.push(['_trackPageview']);
(function() {
var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
})();
</script>
<!-- End Google Analytics -->