---
layout: compute
meet_date: 2019-08-16
topic: "Nanopore data processing"
leaders: "Group Discussion"
---

Please read the following article and come prepared to discuss.  
  
[Rang FJ, Kloosterman WP, De Ridder J (2018). From squiggle to basepair: computational approaches for improving nanopore sequencing read accuracy. Genome Biology 19: 90](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-018-1462-9)  
  
We will also review some of the following tools.  
  
Quality assessment:  
- [NanoPlot](https://github.com/wdecoster/NanoPlot)
- [Poretools](https://poretools.readthedocs.io/en/latest/index.html)
- [BulkVis](https://doi.org/10.1093/bioinformatics/bty841)
- [PycoQC](https://github.com/a-slide/pycoQC)
- [MinIONQC](https://github.com/roblanf/minion_qc)
- [PyPore](https://github.com/rsemeraro/PyPore)  
  
Adapter trimming:  
- [Porechop](https://github.com/rrwick/Porechop)  
  
Mapping:  
- [minimap2](https://github.com/lh3/minimap2)
  
Officially ONT-supported pipelines:   
- [ont-assembly-polish](https://github.com/nanoporetech/ont-assembly-polish)
- [medaka](https://github.com/nanoporetech/medaka)
- [pipeline-structural-variation](https://github.com/nanoporetech/pipeline-structural-variation)
  
Methylation calling:  
- [tombo](https://github.com/nanoporetech/tombo)
